src/main/resources/resource/Arduino/MrlComm/MrlServo.cpp

```diff
@@ -128,6 +128,7 @@ void MrlServo::moveToMicroseconds(int posUs) {
   isMoving = true;
   lastUpdate = millis();
   moveStart = lastUpdate;
+  publishServoEvent(SERVO_EVENT_STARTED);
```

Comment:
Generally we should never publish anything from inside the processCommand call.  Any writing to the serial port should be forbidden here.  it should work. we just shouldn't do it.